kvm/v12.0.0/announcement.md

```diff
@@ -0,0 +1,3 @@
+# :zap: Giant Swarm Release 12.0.0 for KVM :zap:
+
+KVM release 12.0.0 is now available. This is the first Giant Swarm release which includes Kubernetes v1.17. Many core components and default apps have been updated for improved reliability and observability. Further details about changes to individual components can be found by visiting [this link](https://github.com/giantswarm/releases/blob/master/kvm/v12.0.0).
```

Comment:
As per Slack discussion, wdyt @tfussell ?
```suggestion
KVM release 12.0.0 is now available. This is the first Giant Swarm release which includes Kubernetes v1.17. Many core components and default apps have been updated for improved reliability and observability. Further details about changes to individual components can be found by visiting [this link](https://github.com/giantswarm/releases/blob/master/kvm/v12.0.0).

Please reply to this announcement in a thread if you have any feedback or questions.
<!subteam^S0GSG846L> 
```

Reply:
or:
Please help <!subteam^S0GSG846L> improve communication and reply to this announcement in a thread if you have any feedback or questions.

wierd string will be replaced with ProductOwners